handbook/company.md

```diff
@@ -260,6 +200,76 @@ Between overthinking and rushing, there is a [golden mean](https://en.wikipedia.
+
+- One repo keeps all of the relevant work in one place. 
+- One repo means that there is less to get lost. Think of it as if more repos take up a sprawling surface area that becomes unruly to manage and keep up with from day to day. This also makes it easy to find everything.
+- One repo pools GitHub stars to reflect Fleet’s actual presence better.
+- With only one repo, you can consolidate your automation and labels. Otherwise, you have to manage labels and automation across multiple repos, which can drift.
+
+
+### Why organize work in team-based kanban boards?
+It is helpful to have a shared framework for how every team works, plans, and requests things from each other. Fleet's kanban boards are that framework. 3 goals:
+
+- Intake: Give people from anywhere in the world the ability to request something from a particular team (i.e., add it to their backlog).
+- Planning: Give the team's manager and other team members a way to plan the next 3-week iteration of what the team is working on in a world (the board) where the team has ownership and feels confident making changes.
+- Shared to-do list: What should I work on next? Who needs help? What important work is still blocked? Is that bug fix merged yet? When will it be released? When will that new feature ship? What did I do yesterday?
+
+
+### Why a 3-week cadence?
+Fleet product is released every 3 weeks, so everyone in the company is synced up to this same schedule and is always aware of product releases even if they aren’t on the product team..
+- Other companies use a 4-week release cycle, but at Fleet, we like to move a little faster to get more done.
+- Everyone knows when the new release is, so they also know when their work is due.
+
+
+### Why use agile?
+- See [the agile manifesto](https://agilemanifesto.org/).
+- Releasing software iteratively results in it actually working, gets it into the hands of users faster, and makes contributors fitter, happier, and more productive.
+
+
+### Why the emphasis on training?
+Investing in people and providing generous, prioritized training, especially up front, helps contributors understand what is going on at Fleet. This is a prerequisite for helping people make better decisions at work and to feel confident while doing so. Keep in mind those who may feel like outsiders to the osquery (security/startup/IT) space.
+- Creating a culture of helping others results in people feeling more comfortable and confident even if they aren’t familiar with osquery.
+- A sharp focus on training means things are written down.
+
+
+### Why not continuously generate REST API reference docs from javadoc-style code comments?
+- Using Markdown allows anyone to edit Fleet's docs.
+- Generated docs become just as out of date as handmade docs, except since they are generated, they can become more difficult to edit and therefore gated/siloed.
```

Comment:
Sometimes we miss updating the REST API reference docs because they're handmade. Here's an example from this week: https://fleetdm.slack.com/archives/C019WG4GH0A/p1649864136756099?thread_ts=1649863779.331849&cid=C019WG4GH0A

Reply:
Agreed, we need a process here to make sure that happens.  The same thing can happen when they are generated.